src/desktop/apps/artist/server.js

```diff
@@ -47,10 +47,6 @@ app.get("/artist/:artistID*", async (req, res, next) => {
       !user && isExternalReferer && !IS_MOBILE
     res.locals.sd.ARTIST_PAGE_CTA_ARTIST_ID = req.params.artistID
 
-    // While we are rolling out the new page, override the default (`artist`)
```

Comment:
This is the other legit diff/key update (nice job git on generating a nice diff, easy to see straightforward deletions and renames).

We remove the page type override from `new-artist` back to `artist` (cc @joeyAghion any updates to tracking or dashboards needed?). Also use the renamed asset package.

Reply:
I don't think anything is referencing `new-artist` directly, so this is 👍.